day33/BOJ_16918_봄버맨/BOJ_16918_봄버맨_신영제.java

```diff
@@ -0,0 +1,111 @@
+package AlgoStudy;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.StringTokenizer;
+
+public class BOJ_16918봄버맨 {
+	
+	static int R, C, N;
+	   
+	   static char[][] arr;
+	   static int[][] bombTime;
+	   
+	   static int time = 0;
+	   
+	   static int[] dx = {-1,0,1,0};
+	   static int[] dy = {0,1,0,-1};
+	   
+	    public static void main(String[] args) throws IOException {
+	        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+	        StringTokenizer st = null;
+	        
+	        st = new StringTokenizer(br.readLine());
+	         R = Integer.parseInt(st.nextToken());
+	         C = Integer.parseInt(st.nextToken());
+	         N = Integer.parseInt(st.nextToken());
+
+	        arr = new char[R][C];
+	        bombTime = new int[R][C];
+
+	        for (int i = 0; i < R; i++) {
+	            String str = br.readLine();
+	            for (int j = 0; j < C; j++) {
+	                arr[i][j] = str.charAt(j);
+	                if(arr[i][j]=='O'){
+	                    bombTime[i][j] = 3;
+	                }
+	            }
+	        }
+
+	        while(true) {
+	            if(time>=N) break;
+	            time ++;
```

Comment:
```suggestion
                    time++;
```
단항연산자 앞에는 띄어쓰기 삭제해주세요.